### ISSUE TITLE
how to stop: Do you want the application “Chromium.app” to accept incoming network connections? pop up dialog

### ISSUE THREAD

**Issue by alicazi** [go-rod/rod#467]
**Rod Version:** v0.105.5

I'm getting '**Do you want the application “Chromium.app” to accept incoming network connections?**' message with a pop dialog almost on every launch of the browser from rod. Is there any way to get rid from this message. It is really annoying.

I created a certificate and assigned to Google Chrome. App but it didn't resolve the issue

**Comment by ysmood** [comment]
I think you have to trust the browser located here:

https://github.com/go-rod/rod/blob/c00ca2f04b1e68d9d77a87bce59560c7046f601c/lib/launcher/browser.go#L55-L57

**Comment by ysmood** [comment]
I think you have already checked https://github.com/puppeteer/puppeteer/issues/4752

**Comment by alicazi** [comment]
Yes,  I did! I added the certificate and followed the codesign step. It may improve the situation as in some cases I don't see the pop up but it still show up frequently.

**Comment by alicazi** [comment]
I found chromium.app in /Users/myuser/.cache/rod/browser/chromium-884014/chrome-mac/Chromium.app. I tried to codesign the certificate but now I'm getting the following error:
cannot find code object on disk
In subcomponent: /Users/myuser/.cache/rod/browser/chromium-884014/chrome-mac/Chromium.app/Contents/Frameworks/Chromium Framework.framework

**Comment by ysmood** [comment]
I use mac too, when I turn on firewall, I can't reproduce your issue, sorry, not sure how to help. Also the puppeteer issue I posted is still open for years, it might be an unresolved issue.

**Comment by alicazi** [comment]
I finally fixed the issue. This is what I did.
From the following code I found the path to chrome app used by rod:
...
path, _ := launcher.LookPath()
...

The path was pointing to "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome".
So I created the certificate as mentioned in puppeteer/puppeteer#4752 (thanks to @ysmood) and assigned the certificate to "Google Chrome.app". 

**Note: Don't forget to change the trust of the certificate to "Always Trust" because I forgot to do this and the pop up kept showing up here and there.**

_What I can say is: "$HOME/.cache/rod/browser" is not the location of the chrome app that needs the certificate_
